export const getData = async() => {
    try {
        
        
    } catch (error) {
        
    }
}